### PR TITLE
repair http register

### PR DESCRIPTION
**Pull request by lw1243925457**
// Describe your PR here; eg. Fixes #issueNo

Make sure that:

- [ ] You have read the [contribution guidelines](https://dromara.org/en-us/docs/soul/contributor.html).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: soul-register-center/soul-register-client/soul-register-client-http/src/main/java/org/dromara/soul/register/client/http/utils/RegisterUtils.java [hunk 1]
```diff
@@ -42,7 +42,7 @@ private RegisterUtils() {
     public static void doRegister(final String json, final String url, final String rpcType) {
         try {
             String result = OkHttpTools.getInstance().post(url, json);
-            if ("success".equals(result)) {
+            if ("SUCCESS".equals(result)) {
```
Comment (dengliming) [inline soul-register-center/soul-register-client/soul-register-client-http/src/main/java/org/dromara/soul/register/client/http/utils/RegisterUtils.java:45]: Why modify this?

**Review by dengliming — CHANGES_REQUESTED** [review]

File: soul-register-center/soul-register-client/soul-register-client-http/src/main/java/org/dromara/soul/register/client/http/utils/RegisterUtils.java (hunk 1 above)
Comment (lw1243925457) [reply soul-register-center/soul-register-client/soul-register-client-http/src/main/java/org/dromara/soul/register/client/http/utils/RegisterUtils.java:45]: @PostMapping("/springmvc-register")
    public String registerSpringMvc(@RequestBody final SpringMvcRegisterDTO springMvcRegisterDTO) {
        INSTANCE.publishEvent(RpcTypeEnum.HTTP.getName(), springMvcRegisterDTO);
        return "SUCCESS";
    }

Comment (dengliming) [reply soul-register-center/soul-register-client/soul-register-client-http/src/main/java/org/dromara/soul/register/client/http/utils/RegisterUtils.java:45]: ...I suggest that use `org.dromara.soul.admin.utils.SoulResultMessage#SUCCESS` in controller.

Comment (lw1243925457) [reply soul-register-center/soul-register-client/soul-register-client-http/src/main/java/org/dromara/soul/register/client/http/utils/RegisterUtils.java:45]: public String registerSpringMvc(@RequestBody final SpringMvcRegisterDTO springMvcRegisterDTO) {
        INSTANCE.publishEvent(RpcTypeEnum.HTTP.getName(), springMvcRegisterDTO);
        return SoulResultMessage.SUCCESS;
    }

like this?

Comment (dengliming) [reply soul-register-center/soul-register-client/soul-register-client-http/src/main/java/org/dromara/soul/register/client/http/utils/RegisterUtils.java:45]: yes. That's what I mean. thanks.

Comment (lw1243925457) [reply soul-register-center/soul-register-client/soul-register-client-http/src/main/java/org/dromara/soul/register/client/http/utils/RegisterUtils.java]: okay，I have modified.

**Review by yu199195 — APPROVED** [review]

**Pull request merged**